website/source/docs/drivers/docker.html.md

```diff
@@ -688,6 +688,9 @@ options](/docs/configuration/client.html#options):
   disable Nomad from removing a container when the task exits. Under a name
   conflict, Nomad may still remove the dead container.
 
+* `docker.nvidia_runtime`: Defaults to `nvidia`. This option can be used to
```

Comment:
This option allows operators to select the runtime that should be used in order to expose Nvidia GPUs to the container.